apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx

```diff
@@ -66,98 +63,67 @@ export const UsePoolerCheckbox = ({
-                                                  'rounded bg-alternative py-1 px-2 leading-none shadow',
-                                                  'border border-background w-[380px]',
-                                                ].join(' ')}
-                                              >
-                                                <span className="text-xs text-foreground">
-                                                  Pooling mode is currently configured to use
-                                                  session mode. If you want to use both session mode
-                                                  and transaction mode at the same time, change the
-                                                  pooling mode to transaction first in the pooler
-                                                  configuration settings. You can then connect to
-                                                  session mode on port 5432 and transaction mode on
-                                                  port 6543
-                                                </span>
-                                              </div>
-                                            </Tooltip.Content>
-                                          </Tooltip.Portal>
-                                        )}
-                                      </Tooltip.Root>
-                                    </CommandItem_Shadcn_>
-                                    <CommandItem_Shadcn_
-                                      value="session"
-                                      className="cursor-pointer w-full text-foreground"
-                                      onSelect={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                      onClick={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                    >
-                                      Session mode
-                                    </CommandItem_Shadcn_>
-                                  </CommandGroup_Shadcn_>
-                                </CommandList_Shadcn_>
-                              </Command_Shadcn_>
-                              <Separator />
-                              <p
-                                className="px-3 py-2 text-xs text-foreground-light hover:text-foreground cursor-pointer transition"
+                          <DropdownMenu>
+                            <DropdownMenuTrigger asChild>
+                              <Button
+                                type="outline"
+                                className="py-0.5 pr-1.5"
+                                iconRight={<IconChevronDown strokeWidth={1.5} />}
```

Comment:
strokeWidth 1

we don't use anything but 1 for icons

Reply:
just a pref though but doesnt strokeWidth 1 feel too thin?